### ISSUE TITLE
document filter

### ISSUE THREAD

**Issue by DavidBruchmann** [TYPO3-Documentation/t3docs-search-indexer#24]
Having tested the new global search on the documents page I want to add my opinion:
I searched for page = PAGE , can narrow it down by clicking TYPO3 manual (2407), but it's still showing very much.
If the different documents would be shown in a list for filtering, it would be overview and choice in one.

**Comment by DavidBruchmann** [comment]
for clarification:  
![image](https://user-images.githubusercontent.com/999146/126638358-8058efbf-23f9-4eca-afeb-0df9a8ed6747.png)

**Comment by tmotyl** [comment]
@gr0 how this could be solved? some sub-buckets ?

**Comment by gr0** [comment]
yes, that can be done by using sub-buckets on some data. The key point here is what data would be interesting and helpful for narrowing down the search results. But from the comment I can see that you already know what fields could be used. So, here, top level `terms` aggregation and sub-buckets with `terms` aggregation as well. If you can provide an example of a `_source` field from Elasticsearch I can provide some feedback if needed. 

Of course, we could even go further and in sub-buckets use something like `significant_text` or `significant_term` aggregation on the documentation page, but that can turn out wide and not give very good results. It all depends on the data.

**Comment by DavidBruchmann** [comment]
I never know about Elasticsearch or sub-buckets, so help would be much appreciated.

The current filters look like this:

![image](https://user-images.githubusercontent.com/999146/221098086-447638de-f049-4f1a-b93a-d59995a82e04.png)

**Comment by tmotyl** [comment]
related https://github.com/TYPO3-Documentation/t3docs-search-indexer/issues/34
